lib/gutentag.rb

```diff
@@ -10,7 +10,7 @@ def self.dirtier=(dirtier)
   end
 
   def self.normaliser
-    @normaliser ||= Gutentag::TagName
+    @normaliser ||= -> (t){ t.to_s.downcase }
```

Comment:
Can this be changed to the old-school lambda syntax? Also, would rather avoid single-character parameters, so `t` should become `tag`.